tests/models/stablelm/test_modeling_stablelm.py

```diff
@@ -0,0 +1,428 @@
+# coding=utf-8
+# Copyright 2024 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch StableLM model. """
+
+
+import unittest
+
+from parameterized import parameterized
+
+from transformers import StableLmConfig, is_torch_available, set_seed
+from transformers.testing_utils import (
+    require_flash_attn,
+    require_torch,
+    slow,
+    torch_device,
+)
+
+from ...generation.test_utils import GenerationTesterMixin
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ModelTesterMixin, ids_tensor
+from ...test_pipeline_mixin import PipelineTesterMixin
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        AutoTokenizer,
+        StableLmForCausalLM,
+        StableLmForSequenceClassification,
+        StableLmModel,
+    )
+
+
+# Copied from transformers.tests.models.persimmon.test_modeling_persimmon.PersimmonModelTester with Persimmon -> StableLm
+class StableLmModelTester:
+    # Ignore copy
+    def __init__(
```

Comment:
```suggestion

    # Ignore copy
    def __init__(
```

Reply:
Running `make style` reformats the new line between `class ...` and   `# Ignore copy` 😅

Reply:
Ah alright no worries then!